packages/frontend/src/app/page.tsx

```diff
@@ -20,12 +24,15 @@ async function getEntities(): Promise<Entity[]> {
 
 export default async function Home() {
   const entities: Entity[] = await getEntities()
+  const token = getAuthToken()
+  const user = await getUser(token)
+  console.log('user', user)
```

Comment:
Do we want to log?

Reply:
Yes, there are still some unresolved things around auth on the frontend. We can remove them later.